tests/index.js

```diff
@@ -71,5 +72,9 @@ describe('<ContentLoader />:', () => {
     it('`style` is a object and used', () => {
       expect(wrapper.props().style).to.deep.equal({ marginBottom: '10px' })
     })
+
+    it('`animation` is set to false', () => {
+      expect(wrapper.props().animation).to.be.false
+    })
```

Comment:
It's right, but I think in this case you still should test the `true` and if it has `<animate />` at DOM